cmd/pctl/install.go

```diff
@@ -67,7 +68,17 @@ func installCmd() *cli.Command {
 			&cli.StringFlag{
 				Name:  "repo",
 				Value: "",
-				Usage: "The repository to open a pr against. Format is: org/repo-name",
+				Usage: "The repository to open a pr against. Format is: org/repo-name.",
+			},
+			&cli.StringFlag{
+				Name:  "url",
+				Value: "",
+				Usage: "Optional value defining the URL of the profile.",
+			},
+			&cli.StringFlag{
+				Name:  "path",
+				Value: "",
+				Usage: "Value defining the path to a profile when url is provided.",
 			},
```

Comment:
we have an existing `--branch` flag, but thats for `create-pr`, so idk how we want to handle this two different use cases for branchs :cry:

Reply:
True. :/ Uh, I'll find something out. :D